docs/sources/guides/getting_started.md

```diff
@@ -1,49 +1,86 @@
 +++
-title = "Getting Started"
-description = "Getting started with Grafana guide"
+title = "Getting started"
+description = "Guide for getting started with Grafana"
 keywords = ["grafana", "intro", "guide", "started"]
 type = "docs"
 aliases = ["/docs/grafana/latest/guides/gettingstarted"]
 [menu.docs]
-name = "Getting Started"
+name = "Getting started"
 identifier = "getting_started_guide"
 parent = "guides"
-weight = 1
+weight = 100
 +++
 
 # Getting started
 
-This guide will help you get started and acquainted with Grafana. It assumes you have a working Grafana server up and running. If not please read the [installation guide]({{< relref "../installation/" >}}).
+This guide will help you get started and acquainted with Grafana.
 
-## Log in for the first time
+## Install Grafana
 
-To run Grafana, open your browser and go to http://localhost:3000/. 3000 is the default HTTP port that Grafana listens to if you haven't [configured a different port]({{< relref "../installation/configuration/#http-port" >}}).
+This step varies according to your computer operating system. Refer to the instructions for your OS in the [Installation]({{< relref "../installation/_index.md" >}}) section for instructions.
 
-There you will see the login page. Default username is admin and default password is admin. When you log in for the first time you will be asked to change your password. We strongly encourage you to follow Grafana’s best practices and change the default administrator password. You can later go to user preferences and change your user name.
+## Log in for the first time 
+
+1. Open your web browser and go to http://localhost:3000/. `3000` is the default HTTP port that Grafana listens to if you haven’t configured a different port.
+1. On the login page, type `admin` for the username and password.
+1. Change your password. 
+
+> **Note:** We strongly encourage you to follow Grafana best practices and change the default administrator password. Don't forget to record your credentials!
+
+## Add TestData DB data source
+
+1. Click **Add data source**.
+1. Search for or scroll down to TestData DB.
+1. Click **TestData DB** and then click **Save & Test**.
```

Comment:
This is actually not needed, the random walk scenario is already there in a built in data source named "-- Grafana --". But this is still a good thing to have in the guide if someone just wants to play with what visualizations grafana have. But maybe not as a required getting started step.

Reply:
So...how do we get visible results without adding the data source?

Reply:
NVM, figured it out.